tool/tsh/tsh.go

```diff
@@ -954,7 +959,12 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 	// Connect to the span exporter and initialize the trace provider only if
 	// the --trace flag was set.
 	if cf.SampleTraces {
-		provider, err := newTraceProvider(&cf, command, []string{login.FullCommand()})
+		// login only needs to be whitelisted if forwarding to auth
+		var whitelist []string
```

Comment:
```suggestion
		var allowed []string
```

https://www.forbes.com/sites/carolinamilanesi/2021/06/29/the-importance-of-inclusive-language-and-design-in-tech/?sh=ead71d9b1672